spec/spec_helper.rb

```diff
@@ -18,7 +20,7 @@
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
 # Requires factories defined in lib/solidus_jwt/factories.rb
```

Comment:
Can you update the comment with the correct path? I think it's missing `/testing_support`